lms/envs/common.py

```diff
@@ -1355,15 +1355,17 @@
     },
     'style-certificates': {
         'source_filenames': [
-            'certificates/css/main-ltr.css',
+            'sass/certificates/main-ltr.css',
```

Comment:
@feanil, I'd prefer to keep the directory structure we have set up and am fine bending back to the platform's compile location preference. So that means, my ideal call here would be  ``certificates/sass/main-ltr.css``.

I know you made some changes to the scope of directories that our platform asset/sass watchers are aware of. Would that need to be reverted in order to have this happen?